src/server/src/main/java/com/spotify/reaper/storage/CassandraStorage.java

```diff
@@ -539,6 +539,9 @@ private static RepairSegment createRepairSegmentFromRow(Row segmentRow){
     Collection<RepairRun> repairRuns = getRepairRunsForCluster(clusterName);
 
     for(RepairRun repairRun:repairRuns){
+      if(repairRun.getRunState() != RunState.RUNNING){
```

Comment:
nit-pick: spaces, so like `if (repairRun.getRunState() != RunState.RUNNING) {`

Reply:
wonder if making lines 539-550 into 
```

    getRepairRunsForCluster(clusterName)
            .stream()
            .filter(r -> r.getRunState() != RunState.RUNNING)
            .forEach(...);
```
would be more elegant?

Reply:
I changed the spacing. Originally, I simply made it look like the rest of the file. Is there a style-guide for Cassandra Reaper?

Regarding the functional approach: I like it, but all of the rest of the code is written in a imperative way. Therefore I think translating everything into a more functional style should be a separate change, that is not related to this bugfix.

Reply:
@smarsching we've got an issue open to discuss using a formatting tool, #182.